packages/cli/src/UserManagement/routes/users.ts

```diff
@@ -168,6 +173,11 @@ export function usersNamespace(this: N8nApp): void {
 							email,
 						});
 						resp.error = `Email could not be sent`;
+					} else {
+						void InternalHooksManager.getInstance().onUserTransactionalEmail({
+							user_id: id!,
+							message_type: 'New user invite',
+						});
```

Comment:
we should probably swap the logic here and check `if (result.success) { Internalhooks } else { error } `